src/react-components/src/components/02-molecules/Cards/SearchResultItem.tsx

```diff
@@ -0,0 +1,88 @@
+/* MC-25 Search Results Item */
+import * as React from "react";
+import bem from "../../../utils/bem";
+import IconLink from "../../01-atoms/Links/IconLink";
+import Heading from "../../01-atoms/Text/Headings/Heading";
+import EditionCard from "./EditionCard";
+import UnderlineLink from "../../01-atoms/Links/UnderlineLink";
+
+export type EditionInfo = {
+  editionYearHeading: JSX.Element,
+  publisherAndLocation: string,
+  coverUrl: string,
+  language: string,
+  license: string,
+  readOnlineLink: string,
+  downloadLink: string
+};
+
+export interface SearchResultItemProps {
+  id: string;
+
+  resultIndex: number;
+
+  modifiers?: [];
+  blockName?: string;
+
+  headingContent: JSX.Element;
+
+  subtitleText: string;
+
+  authorLinkElement: JSX.Element;
+
+  editionInfo: EditionInfo;
+  editionsLinkElement: JSX.Element;
+}
+
+export default class SearchResultItem extends React.Component<SearchResultItemProps, {}> {
+  constructor(props: SearchResultItemProps) {
+    super(props);
+  }
+
+  render(): JSX.Element {
+    const { id,
+      resultIndex,
+      modifiers, blockName, headingContent,
+      subtitleText,
+      authorLinkElement,
+      editionInfo, editionsLinkElement} = this.props;
+    const baseClass = "search-result-item";
+
+    try {
+      React.Children.only(headingContent as React.ReactElement);
+    } catch (e) {
+      const children = React.Children.map(this.props.children, child =>
+        (child as JSX.Element).type);
+      // Catching the error because React's error isn't as helpful.
+      throw new Error(`Please only pass one child into SearchResultItem, got ${children.join(", ")}`);
+    }
+
+    // TODO: Decide whether this needs to be in DS, and write/find utilities for us to count text within child components
```

Comment:
Didn't know you couldn't count text in children!

Reply:
There's this utility that exists: https://github.com/fernandopasik/react-children-utilities 
But I'll need time to figure out whether we want to re-implement the text counting or include the package from them.  Probably good to put in a DS ticket

Reply:
https://github.com/NYPL/nypl-design-system/projects/2#card-31474290